lib/AST/CMakeLists.txt

```diff
@@ -72,6 +77,7 @@ add_swift_library(swiftAST STATIC
     clangLex
     clangAPINotes
     clangBasic
+    ${system_libs}
```

Comment:
This doesnt really make sense.  This is a static library, so the link here is meaningless except as an `INTERFACE`.